test/helpers/index.ts

```diff
@@ -15,16 +15,22 @@ export const generateConfig = (
     fee_income_recipients?: { [key: string]: string };
     bond_reward_recipients?: { [key: string]: string };
   } = {}
-): Config => {
+): BreadcrumbsConfiguration => {
   return {
     baking_address: args.baking_address || DEFAULT_BAKER,
-    default_fee: args.default_fee || "5",
+    default_fee: !isNaN(Number(args.default_fee))
```

Comment:
I'd say the bang is adding an unnecessary cognitive load. Suggest: 

```javascript
    default_fee: isNaN(Number(args.default_fee))?
      ? 0
      :  Number(args.minimum_payment_amount),
```

Reply:
Same for ones below.

Reply:
fixed